src/documentation/dsls/sql.md

```diff
@@ -14,97 +14,141 @@ This page describes the implementation of Beam SQL, and how to simplify a Beam p
 
 ## 1. Overview {#overview}
 
-SQL is a well-adopted standard to process data with concise syntax. With DSL APIs (currently available only in Java), now `PCollection`s can be queried with standard SQL statements, like a regular table. The DSL APIs leverage [Apache Calcite](http://calcite.apache.org/) to parse and optimize SQL queries, then translate into a composite Beam `PTransform`. In this way, both SQL and normal Beam `PTransform`s can be mixed in the same pipeline.
+SQL is a well-adopted standard to process data with concise syntax. With DSL APIs (currently available only in Java), now `PCollections` can be queried with standard SQL statements, like a regular table. The DSL APIs leverage [Apache Calcite](http://calcite.apache.org/) to parse and optimize SQL queries, then translate into a composite Beam `PTransform`. In this way, both SQL and normal Beam `PTransforms` can be mixed in the same pipeline.
```

Comment:
Drive by comment -- if you end up making another commit for this PR, could we axe the * TOC {:toc} lines at the top of the page? We didn't have the right nav at the time this was added. Due to the bolding of a couple of the deeply nested sections, it looks a bit strange also.

Reply:
Removed